lgc/patch/PatchBufferOp.cpp

```diff
@@ -1673,12 +1666,14 @@ void PatchBufferOp::fixIncompletePhis() {
     assert(isa<UndefValue>(phiNode->getIncomingValueForBlock(incomingBlock)));
     assert(phiNode->getType()->isVectorTy() || phiNode->getType()->isPointerTy());
 
-    if (phiNode->getType()->isVectorTy())
+    Replacement pointer = getRemappedValue(incoming);
```

Comment:
Not quite no functional change - presumably you intended it to handle a null here better (at least AFAIK getRemappedValue will invoke getRemappedValueOrNull if necessary?)

Reply:
Yes getRemappedValue handles nullptr properly but in practice I don't think that is relevant here, because `incoming` will always be an Instruction, otherwise we wouldn't have added this phi input to the incomplete phis list in the first place. So using getRemappedValue here is really just a handy way of avoiding an explicit cast from Value to Instruction.